homeassistant/components/media_player/kodi.py

```diff
@@ -364,6 +378,36 @@ def ws_loop_wrapper():
         self.hass.loop.create_task(ws_loop_wrapper())
 
     @asyncio.coroutine
+    def async_added_to_hass(self):
+        """Script creation for the turn on/off config options."""
+        if self._turn_on_action is not None:
+            self._turn_on_action = script.Script(
+                self.hass, self._turn_on_action, "Kodi Turn ON action script")
+        if self._turn_off_action is not None:
+            # Deprecation check
+            if isinstance(self._turn_off_action, str):
```

Comment:
Do you mean to disable Kodi for almost everyone with the update? I respectfully disagree. IMHO, the warning will be sufficiently explicit for the user to see it and will made the change easily by copy/pasting from the log, everything without the need to see a bad Kodi setup and restart after the obligated analysis. These are very _precious_ minutes, for me at last.

I know the # of users is not in the equation here to decide this, but, _off-topic-ing_ a little only because curiosity, do we now already some big numbers about the components and platforms penetration, from the stats collected by the `updater` component? I know it's so early for this, but I'm a very curious guy ;-)